client/lib/signup/actions.js

```diff
@@ -27,6 +26,7 @@ const SignupActions = {
 	},
 
 	submitSignupStep( step, errors, providedDependencies ) {
+		analytics.tracks.recordEvent( 'calypso_signup_actions_submit_step', { step: step.stepName, ...providedDependencies } );
```

Comment:
I know this is a very, very minor nitpick, but if you revert the order:

```
{ ...providedDependencies, step: step.stepName }`
```

You can make sure that a future dependency named `step` (unlikely, but not impossible) will not break the event.